app/design/frontend/Magento/blank/Magento_Newsletter/web/css/source/_module.less

```diff
@@ -90,3 +90,12 @@
         }
     }
 }
+
+//
+//  Tablet
+//  _____________________________________________
+@media only screen and (max-width: @screen__m) {
+  .block.newsletter {
+    width: 42%;
+  }
+}
```

Comment:
Hi, @dipti2jcommerce!

Please add an empty line in the end of the file

Thank you!

Reply:
Hi @zanilee ,

I added empty line in the end of the file.
please check.

Thank,
Dipti